### PR TITLE
Spec out benchmark tasks

### PR DESCRIPTION
**Pull request by manzt**
@ngehlenborg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,20 @@ So far, we have a list at the top of `js/App.js` called `transitionViewStates` t
 ### Data
 
 Right now only the standard Vanderbilt MxIF Kidney OME-ITFF images are used (tile size 512), with one copy on s3 and GCS. Eventually, we will want to test different tile sizes, file formats (i.e Zarr), local and remote files, number of channels, screen size (mobile vs browser), and all the combinations (on both HTTP and HTTP2).
+
+
+#### Tasks
+- tilesize vs. viewport size:
+  - `tilesizes`: 256x256, 512x512, 1024x1024 
+  - `viewport sizes`: Mobile, Tablet, Desktop, Widescreen?
```
Comment (ilan-gold) [inline README.md]: What is "Widescreen?" Like Nils' TV?

File: README.md [hunk 2]
```diff
@@ -21,3 +21,20 @@ So far, we have a list at the top of `js/App.js` called `transitionViewStates` t
...
+  - Expected results: larger tiles for bigger displays will generally be better; smaller tiles faster to load.
+- number of channels (1-6):
+  - Expected results: fewer channels, faster loading.
+- HTTP1 vs HTTP2.
+  - Toggle on and off HTTP1/2 clientside for 1.) GCS bucket, 2.) Custom server supporting HTTP/2.
+  - Expected results: likely doesn't matter for commercial storage; substantially better for in house server.
+- Local vs Remote:
```
Comment (ilan-gold) [inline README.md:36]: When I said "Local" vs "Remote," I just meant a file server but I agree that it is hard to benchmark.

**Review by ilan-gold — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ngehlenborg) [reply README.md]: It might be better to define these in terms of pixel dimensions rather than somewhat under-defined device types.

**Review by ngehlenborg — APPROVED** [review]
Let's used pixel-based view port sizes but otherwise the plan sounds good.

File: README.md (hunk 1 above)
Comment (manzt) [reply README.md]: updated

**Comment by ngehlenborg** [comment]
Does this still need to get merged or can we close this?

**Comment by manzt** [comment]
This is not the benchmark repo. This repo should probably be archived. https://github.com/hms-dbmi/viv-tile-benchmark

**Pull request closed without merging**